test/spec/index.js

```diff
@@ -56,6 +74,8 @@ describe('Clay', function() {
 
     it('handles the "showConfiguration" event if autoHandleEvents is not false',
     function() {
+      this.timeout(10000); // 10 seconds
```

Comment:
Adding a log line at every step didn't shed any light. It looks like the test runs through all the instructions but still freaks out. I'm okay with the timeout being the solution.